swagger-spec/collection_provider/detail.yaml

```
@@ -0,0 +1,102 @@
+# /provider/collections/{collection_id}/
+get:
+  summary: Collections Providers Detail
+  description: >-
+
+    This returns a single Collections Provider entity.
+
+    #### Permissions
+
+    This information is public.
+
+    #### Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+    The `data` key contains an array of nodes ids.
```

Copy-pasting the wording from other detail endpoints:

```suggestion
   Returns a JSON object with a `data` key containing the representation of the requested collection provider object, if the request is successful.
```